icarus/src/main/dropbox/repositories/CursorRepository.ts

```diff
@@ -3,18 +3,18 @@ import { dropboxAccountId, cursor } from "../Api";
 
 export class CursorRepository {
 
-  constructor(readonly dynamo: DynamoClient, readonly tablename: string) {}
+  constructor(readonly dynamo: DynamoClient) {}
 
   async fetchCursor(accountId: dropboxAccountId): Promise<string> {
-    const result = await this.dynamo.get(this.tablename, {
+    const result = await this.dynamo.get("dropbox_cursors", {
```

Comment:
The only thing is now each table name is defined in two places: serverless.yml and its repository.
Not a massive problem having few tables in total